packages/common/index.ts

```diff
@@ -601,6 +605,11 @@ export type BarChartOutputType = {
   yAxisName?: string;
 };
 
+export type SimpleTableOutputType = {
+  dataSource: any[];
```

Comment:
We wanna avoid using `any` where possible, since these are Ant Design props could we see if there's a built in type for `dataSource` and `columns`? Or to declare a new one